pkg/cmd/roachtest/versionupgrade.go

```diff
@@ -511,9 +514,33 @@ func stmtFeatureTest(
 // the artifacts. The test will fail on purpose when it's done with instructions
 // on where to move the files.
```

Comment:
can we detail the makeFixtureVersion common case?

Reply:
not exactly sure what you mean, but I rewrote the comment. Hope that helped :-)